3_ProtectionProfile/BiocPP.adoc

```diff
@@ -410,57 +410,47 @@ For consistency in language throughout this document, referring to a “lower”
 
 . Technical limitation
 +
-Although different modalities are available for the mobile biometric verification, all modalities may not achieve the same level of accuracy. For modalities that have different target of error rates, ST author may iterate the requirement to set appropriate error rates for each modality.
+Although different modalities are available for the biometric verification, all modalities may not achieve the same level of accuracy. For modalities that have different target of error rates, ST author may iterate the requirement to set appropriate error rates for each modality.
 
 . Number of test subjects required for the performance testing
 +
 Target error rates defined in SFR shall be evaluated based on <<SD>>. Normally the target error rates will directly influence the size of the test subject, the time and cost of the testing. <<SD>> describes how those error rates should be evaluated in an objective manner.
 
-==== FIA_MBV_EXT.2 Quality of biometric samples for mobile biometric verification [[FIA_MBV_EXT.2]]
+==== FIA_MBV_EXT.2 Quality of biometric samples for biometric verification [[FIA_MBV_EXT.2]]
 
 *FIA_MBV_EXT.2.1* The TSF shall only use samples of sufficient quality to verify the user.
 
 *Application Note {counter:remark_count}*:: ST author may refine “sufficient quality” to specify quality standards if the TOE follows such standard.
 
-==== FIA_MBV_EXT.3 Presentation attack detection for mobile biometric verification [[FIA_MBV_EXT.3]]
-
-*FIA_MBV_EXT.3.1* The TSF shall prevent use of artificial presentation attack instruments from being successfully verified.
-
-*Application Note {counter:remark_count}*:: This requirement is only applicable to mobile biometric verification. PAD for biometric enrolment is an optional requirement.
-
-*Application Note {counter:remark_count}*:: Artificial PAIs that the TOE shall prevent and relevant criteria for its security relevant error rates for each type of PAI is defined in <<SD>>.
-
 === Protection of the TSF (FPT)
 ==== FPT_BDP_EXT.1 Biometric data processing [[FPT_BDP_EXT.1]]
 
 *FPT_BDP_EXT.1.1* The TSF shall process any plaintext biometric data used to generate templates and perform sample matching within the security boundary of the secure execution environment.
 
-*Application Note {counter:remark_count}*:: The Consistency Rationale in the appropriate PP-Configuration explains how the TOE in cooperation with its environment shall protect biometric data in detail.
+*Application Note {counter:remark_count}*:: The Consistency Rationale in the <<PPC-MDF>> explains how the biometric system in cooperation with the mobile device shall protect biometric data in detail.
```

Comment:
Here is the main place I'm talking about having specific references. The original states "appropriate PP-Configuration" while the update specifically points to the "PPC-MDF" configuration. This would mean the GPOS would not be able to match up without a new PPM update to add it here, and was the point of changing the text to be more generic and point to the appropriate PPC.

Reply:
OK, I didn't know your intention. I will update the text as you suggested. but how do you think about base PP identification/consistency rationale in the PP-Module?